docs/decisions/0014-no-new-apps.rst

```diff
@@ -0,0 +1,27 @@
+14. Justifying new Django applications in edx-platform
+
+Status
+------
+In Review
+
+Context
+-------
+The Open edX platform is moving toward a more modular architecture. The goal is to transition from a monolithic application in edx-platform to one in which this repository represents a small, stable core with volatility pushed into extensions and plugins. To that end, much of the original edx-platform repository has been split out into micro-frontends, other microservices, plugins, and libraries. However, there are still a number of optional and non-core Django applications within edx-platform and new ones continue to be added.
+
+Decision
+--------
+From the adoption of this ADR, no new Django applications should be added into the edx-platform repository without an accompanying ADR explaining why the application cannot or should not be separated out.
```

Comment:
1. I wonder if we should add a little more text here about the options for "separated out".
2. Whether here, or elsewhere in the ADR, can we provide a link to the plugin docs: https://github.com/openedx/edx-django-utils/blob/master/edx_django_utils/plugins/README.rst?
3. Somewhere, could we mention that plugins are especially useful for features that are specific to a particular deployment, and not expected to be reused throughout the community.